src/github.com/getlantern/fronted/test_support.go

```diff
@@ -0,0 +1,30 @@
+package fronted
+
+import (
+	"crypto/x509"
+	"testing"
+
+	"github.com/getlantern/keyman"
+)
+
+// ConfigureForTest configures fronted for testing using default masquerades and
+// certificate authorities.
+func ConfigureForTest(t *testing.T) {
```

Comment:
This also makes the other test files nice and short again, so when we make future changes to them, GitHub will be able to display a proper diff.